core/org.osate.aadl2.model.editor/.gitignore

```diff
@@ -1,3 +1 @@
-/bin
-/target
```

Comment:
Was this intentional? It doesn't make sense to remove `target` from `.gitignore`.

Reply:
That was by accident.